Userland/Libraries/LibGL/SoftwareGLContext.cpp

```diff
@@ -732,6 +732,36 @@ void SoftwareGLContext::gl_disable(GLenum capability)
         m_rasterizer.set_options(rasterizer_options);
 }
 
+void SoftwareGLContext::gl_gen_textures(GLsizei n, GLuint* textures)
+{
+    if (n < 0) {
```

Comment:
should this be `n <= 0` instead? is a zero value here valid?

Reply:
also what about a null check for `textures` ?

Reply:
Nope. If `n == 0`, nothing happens. This will just fall through. 

![image](https://user-images.githubusercontent.com/4768900/118396712-0f1e6f80-b694-11eb-9542-a865a2fd633e.png)

Reply:
so when n is zero does `m_name_allocator.allocate(n, textures);` work as expected?

Reply:
Yes.

```
for (auto i = 0; i < 0; ++i)
```